dropwizard-client/src/main/java/io/dropwizard/client/DropwizardApacheConnector.java

```diff
@@ -70,11 +70,22 @@
      */
     private final boolean chunkedEncodingEnabled;
 
+    /**
+     * Is the entity in a POST HTTP request can be reloaded from the input, so the request can be retried.
+     */
+    private final boolean isEntityRepeatable;
+
     public DropwizardApacheConnector(CloseableHttpClient client, RequestConfig defaultRequestConfig,
                                      boolean chunkedEncodingEnabled) {
+        this(client, defaultRequestConfig, chunkedEncodingEnabled, !chunkedEncodingEnabled);
+    }
+
+    public DropwizardApacheConnector(CloseableHttpClient client, RequestConfig defaultRequestConfig,
+                                     boolean chunkedEncodingEnabled, boolean isEntityRepeatable) {
```

Comment:
But the issue is that the connector is set up on the client level, not on the request level. It's not possible to have different connectors for different requests. 

If we want to customize it on the request basis, I think we need to go in the direction of handling `ClientProperties` - the same way as it's done for timeouts.

Reply:
I'm not sure if I understand this. Its okay to setup the connector on client level. It should handle the requests more flexible. Therefore it can be parametrized with a predicate which decides if a entity of  a request can be repeated or not.

Reply:
Could you give an example how does it suppose to look with the predicate? AFAIK, the only possible way to customize a request in Jersey is to use client properties, e.g.:
```java
client.target(testUri + "/long-running")
   .property(ClientProperties.READ_TIMEOUT, 5000)
   .request()
   .get()
```

Reply:
In the mean time I did further testing. I think its not possible to repeat a request with body in chunked mode, because the `writeEntity` of Jersey's `ClientRequest` can't be only called once. That means making the `JerseyRequestHttpEntity.isRepeatable()` returning `true` will cause failures. What we were trying to achieve is to prevent  entity buffering because we have very large request entities. I'll try to find a better solution.

Reply:
I got it. I guess it makes sense to provide the user with the ability to control which requests they want to see as repeatable.